### DISCUSSION TITLE
Unable to set CannedACL

### DISCUSSION THREAD

**Discussion by WellitonToledo** [jchristn/Blobject#11] [Q&A]
Hi! 

First of thank you for BlobHelper it is delicous of use.

I can't find a way to set permission when write to blob. For exemple:
CannedACL = S3CannedACL.PublicRead

Has a way to do that?

Regards

**Comment by jchristn** [comment]
Hi and thanks for your kind words!  The library only focuses on read, wrote, delete, and existence operations. Anything related to ACLs would have to be handled outside of the library. Cheers
